packages/typespec-client-generator-core/test/package.test.ts

```diff
@@ -2199,20 +2199,28 @@ describe("typespec-client-generator-core: package", () => {
           name: string;
         }
 
-        @armResourceOperations
-        interface MyInterface {
-          get is ArmResourceRead<MyModel>;
+        namespace MyClient {
+          interface Operations extends Azure.ResourceManager.Operations {}
+
+          @armResourceOperations
+          interface MyInterface {
+            get is ArmResourceRead<MyModel>;
+          }
```

Comment:
Updated test cases to add another operation group which doesn't have `subscriptionId` so that the bug could be catched.